example/scalalib/native/4-common-config/build.mill

```diff
@@ -0,0 +1,71 @@
+// This example shows some of the common tasks you may want to override on a
+// `{language}Module`: specifying the `releaseMode`, `nativeLinkStubs`
+// `nativeIncrementalCompilation, `nativeLinkingOptions` and `nativeWorkdir`.
```

Comment:
`{language}Module` is wrong. Please us `./mill dist.fastPages` and review the documentation yourself before take another pass

Reply:
Noted